test/jdk/java/awt/TrayIcon/ActionEventTest/ActionEventTest.java

```diff
@@ -48,27 +44,25 @@
 import java.awt.image.BufferedImage;
 
 public class ActionEventTest {
-
-    Image image;
     TrayIcon icon;
     Robot robot;
     boolean actionPerformed;
 
     public static void main(String[] args) throws Exception {
         if (!SystemTray.isSupported()) {
             System.out.println("SystemTray not supported on the platform." +
-                " Marking the test passed.");
+                    " Marking the test passed.");
```

Comment:
Perhaps, throwing `jtreg.SkippedException` is a better option.

Now skipped tests are distinguished from passed ones.